react-blogs/alifm-2843/react-blog/src/styles/base.scss

```
@@ -0,0 +1,115 @@
+@use "variables";
+
+body {
+    font-family: Arial;
+    color: variables.$mine-shaft;
+    background-color: variables.$alice-blue;
+    color: rgba(0, 40, 126, 0.882);
+  }
+
+  a {
+    color: rgba(0, 81, 255, 0.882);
+    font-weight: bold;
+}
+
+a:hover {
+    background: variables.$navy-blue;
+    color: variables.$white;
+}
+  
+  .container, .card {
+    display: flex;
+    flex-direction: column;
+    border: 1px solid variables.$regent-st-blue;
+    padding: 20px;
+    border-radius: 8px;
+    
+  }
+
+  section {
+    box-shadow: rgba(100, 100, 111, 0.2) 0px 7px 29px 0px;
+
+  }
+  
+  img {
+    width: 100%;
+    height: 100%;
+    border-radius: 4px;
+    box-shadow: rgba(0, 0, 0, 0.35) 0px 5px 15px;
+    
+  }
+  
+  input[type="search"] {
+    padding: 8px 12px;
+    border-radius: 4px;
+    border: 1px solid variables.$regent-st-blue;
+    width: 400px;
+    background: variables.$link-water;
+    color: rgba(0, 81, 255, 0.882);
+  }
+
+  input::placeholder{
+    color: rgba(0, 81, 255, 0.882);
+  }
+  
+  nav {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    font-size: 12px;
+    border: 1px solid variables.$regent-st-blue;
+    padding: 20px;
+    border-radius: 8px;
+    margin-bottom: 3em;
+  //   grid-column: 1 /4;
+  //   grid-row: 1 / 2;
+  }
+  
+  ul {
+    list-style: none;
+    padding: 0;
+    display: flex;
+    gap: 20px;
+  }
+
+
+  .container {
+    display: grid;
+    grid-template-columns: repeat();
+    grid-template-rows: auto  auto ;
+    gap: 1.5em;
+   margin: 0 1em;
+  }
+  
+  
+
+  h1 {
+    font-size: 1em;
+}
+  
+  #beach {
+    grid-column: 1/4;
+  }
```

```suggestion
  .card:first-of-type {
  grid-column: 1/4;
  }
```
Con esto no necesitarias una id, podrias eliminar el componente BigFirstCard y podrias añadir el contenido de Beach Destinations en tu array de articles

---

Perfecto, genial! Muchas gracias.